packages/commonwealth/client/scripts/state/api/topics/createTopic.ts

```diff
@@ -3,11 +3,13 @@ import axios from 'axios';
 import Topic from 'models/Topic';
 import app from 'state';
 import { ApiEndpoints, SERVER_URL, queryClient } from 'state/api/config';
+import { TopicForm } from 'views/pages/CommunityManagement/Topics/Topics';
 import { userStore } from '../../ui/user';
 
-interface CreateTopicProps {
+interface CreateTopicProps extends TopicForm {
+  // TODO remove those types below when the CreateTopicSectionOld is removed
```

Comment:
lets add a cleanup ticket reference here

Reply:
actually after resolving conflicts, this is not a case anymore as we switched to trpc